src/main/java/com/thealgorithms/maths/HeronsFormula.java

```diff
@@ -1,18 +1,26 @@
 package com.thealgorithms.maths;
 
 /**
+ * Wikipedia for HeronsFormula => https://en.wikipedia.org/wiki/Heron%27s_formula
  * Find the area of a triangle using only side lengths
  */
 
 public class HeronsFormula {
```

Comment:
We can make it a proper _utility class_:

```suggestion
public final class HeronsFormula {
    private HeronsFormula() {
    }
```